### DISCUSSION TITLE
Property for multilingual site name

### DISCUSSION THREAD

**Discussion by sinc13** [microsoft-search/pnp-modern-search#3614] [Q&A]
On our search results page, for each search result we are trying to display the site name of the site that result is coming from. We would also like to show the site name in the search filters webpart. Right now I've mapped a RefinableString to ows_sitename and it's working correctly on our English page. On the French page, however, it is displaying the English site name instead of the translated site name. Is there a property I can use to get the French site names?

**Comment by wobba** [comment]
List/Library/Site names etc in SharePoint are always indexes with the label of the default language for the site. Nothing we can do to overcome this.
